### PR TITLE
 Allow configuring SecRequestBodyAccess and SecResponseBodyAccess

### PR DESCRIPTION
**Pull request by Vincevrp**

### CODE REVIEW

**Comment by Vincevrp** [comment]
@david22swan tests have been added :)

File: spec/classes/mod/security_spec.rb [hunk 1]
```diff
@@ -79,13 +79,17 @@
                 audit_log_type: 'Concurrent',
                 audit_log_storage_dir: '/var/log/httpd/audit',
                 secdefaultaction: 'deny,status:406,nolog,auditlog',
+                secrequestbodyaccess: 'On',
```
Comment (david22swan) [inline spec/classes/mod/security_spec.rb:82]: Minor change, but for the test's could you set the values to something other than the default?

**Review by david22swan — CHANGES_REQUESTED** [review]
LGTM except for on small thing

File: spec/classes/mod/security_spec.rb (hunk 1 above)
Comment (Vincevrp) [reply spec/classes/mod/security_spec.rb]: Changed the values and rebased

**Review by david22swan — APPROVED** [review]
LGTM
Thanks for getting back on the change :)

**Pull request merged**